README.md

```diff
@@ -1 +1,29 @@
-# vips
\ No newline at end of file
+# vip
+
+### Compound Fork Commit
+
+https://github.com/compound-finance/compound-protocol/tree/a3214f67b73310d547e00fc578e8355911c9d376
+
+### Prerequisites
+
+- NodeJS - 12.x
```

Comment:
Can we use a newer version of node? Node LTS is 18. Most repose have the node engine set to 16